client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -75,7 +75,7 @@ class SelectIpsTag extends Component {
 		return this.state.ipsTagList
 			.filter(
 				( hint ) =>
-					this.state.currentQuery && startsWith( hint.tag, toUpper( this.state.currentQuery ) )
+					this.state.currentQuery && startsWith( hint.tag, this.state.currentQuery.toUpperCase() )
```

Comment:
You're going to be revisiting this line soon for `startsWith`, but I assume you want to handle these one at a time 🙂

Reply:
Yep, one at a time. And `startsWith` has many usages, so definitely worth addressing in separate PR(s).